trezorlib/client.py

```diff
@@ -37,6 +37,7 @@
 from .coins import coins_slip44
 from .debuglink import DebugLink
 from .protobuf import MessageType
+from . import stellar as stellar
```

Comment:
drop the `as stellar` please, it's superfluous.
also please move it together with the other `from . import` statements